modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -0,0 +1,176 @@
+With xref:manage:tiered-storage.adoc[Tiered Storage] enabled, you can restore data from a failed cluster (source cluster), including its metadata, onto a new cluster (target cluster). This is a simpler and cheaper alternative to active-active replication, for example with xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]. Use this recovery method to restore your application to the latest functional state as quickly as possible.
+
+CAUTION: Whole cluster restore is not a fully-functional disaster recovery solution. It does not provide snapshot-style consistency. Some partitions in some topics will be more up-to-date than others. Committed transactions are not guaranteed to be atomic.
+
+NOTE: If you need to restore only a subset of topic data, consider using xref:manage:tiered-storage.adoc#remote-recovery[remote recovery] instead of a whole cluster restore.
+
+The following metadata is included in a whole cluster restore:
+
+* Topic definitions. If you have enabled Tiered Storage only for specific topics, topics without Tiered Storage enabled will be restored empty.
+* Users and access control lists (ACLs).
+* xref:manage:schema-reg/schema-reg-overview.adoc[Schemas]. To ensure that your schemas are also archived and restored, you must also enable Tiered Storage for the `_schemas` topic.
+* The xref:develop:consume-data/consumer-offsets.adoc[consumer offsets topic].
```

Comment:
It's somewhat implied from the "CAUTION" blurb above, but it may also be worth mentioning somewhere (maybe here?):

Upon restoring, because we don't provide snapshot style consistency across all partitions, some restored committed consumer offsets may be truncated to a lower value than what was in the original cluster, in order to keep offsets at or below the partitions' restored high watermarks.